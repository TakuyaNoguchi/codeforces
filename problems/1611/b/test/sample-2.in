10
29 11
33 29
19 11
13 17
31 14
3 38
34 5
16 13
34 34
32 14
