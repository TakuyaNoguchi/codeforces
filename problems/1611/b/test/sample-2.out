10
15
7
7
11
3
5
7
17
11
